src/proto/single/cmd.rs

```diff
@@ -1,14 +1,16 @@
-use super::Job;
-use failure::Error;
+use super::{Error, Job};
 use std::io::prelude::*;
 
+/// Default Error type, aliased in module for convenience.
+type Result<T> = std::result::Result<T, Error>;
```

Comment:
I'm not sure I see much value in this type alias. It increases the size of the diff, and means devs now need to look up the alias if editing a given function. It also means we sometimes have to qualify `std::result::Result` below, which isn't ideal.

Reply:
(same for other files)